This bug has been reported as NTP community bug 3103 zsave_warn format string
used without arguments.
http://bugs.ntp.org/show_bug.cgi?id=3103
This patch may be removed after upgrading to a version of NTP that has
bug 3103 fixed.

--- sntp/libopts/save.c	2018-08-20 19:31:19.439321123 +0000
+++ sntp/libopts/save.c	2018-08-20 19:31:44.798573570 +0000
@@ -128,7 +128,7 @@ find_dir_name(tOptions * opts, int * p_f
         }
 
         if (pzEnv == NULL) {
-            fprintf(stderr, zsave_warn, opts->pzProgName);
+            fprintf(stderr, zsave_warn, opts->pzProgName, "\"\"");
             fprintf(stderr, zNotDef, pzDir);
             return NULL;
         }
@@ -180,7 +180,7 @@ find_file_name(tOptions * opts, int * p_
          */
         if (errno != ENOENT) {
         bogus_name:
-            fprintf(stderr, zsave_warn, opts->pzProgName);
+            fprintf(stderr, zsave_warn, opts->pzProgName,pzDir);
             fprintf(stderr, zNoStat, errno, strerror(errno), pzDir);
             if (free_dir_name)
                 AGFREE(pzDir);
@@ -234,7 +234,7 @@ find_file_name(tOptions * opts, int * p_
          */
         if (stat(pzDir, &stBuf) != 0) {
             if (errno != ENOENT) {
-                fprintf(stderr, zsave_warn, opts->pzProgName);
+                fprintf(stderr, zsave_warn, opts->pzProgName, pzDir);
                 fprintf(stderr, zNoStat, errno, strerror(errno),
                         pzDir);
                 AGFREE(pzDir);
@@ -531,7 +531,7 @@ open_sv_file(tOptions * opts)
 
         fp = fopen(pzFName, "w" FOPEN_BINARY_FLAG);
         if (fp == NULL) {
-            fprintf(stderr, zsave_warn, opts->pzProgName);
+            fprintf(stderr, zsave_warn, opts->pzProgName, pzFName);
             fprintf(stderr, zNoCreat, errno, strerror(errno), pzFName);
             if (free_name)
                 AGFREE(pzFName);
